packages/postcss-reduce-initial/data/toInitial.json

```diff
@@ -21,7 +21,6 @@
   "mask-clip": "border-box",
   "mask-mode": "match-source",
   "mask-origin": "border-box",
-  "mask-repeat": "no-repeat",
```

Comment:
I think the previous fix for this was incorrect because if you change `mask-repeat` to `repeat` then you also need to move it to `fromInitial` since now the value is shorter than `initial`.